.github/workflows/tests.yaml

```diff
@@ -55,6 +55,22 @@ jobs:
         run: |
           sudo snap install shellcheck
           make test-shellcheck
+      - name: Install doc framework
```

Comment:
You shouldn't need these steps anymore since you now have https://github.com/canonical/rockcraft/pull/300/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R141-R143